resources/instructions/createSObjectLwcQuickActions.html

```diff
@@ -55,6 +55,10 @@ <h1>Create sObject LWC Quick Actions</h1>
             </tbody>
         </table>
 
+        <div id="errorDiv">
```

Comment:
nit: It would be nice to set a text styling here for the error messages so as to grab user's attention. We could start with something simple like `<div id="errorDiv" style="color:crimson;">` and let UX fine tune it later.

Reply:
Good idea, haven't received designs from product or UX, so just left it pretty bland, but I like your idea. Here it is:

<img width="751" alt="Screenshot 2024-02-20 at 1 30 21 PM" src="https://github.com/salesforce/salesforcedx-vscode-mobile/assets/2682021/1a3cc0e1-dfb3-4ddf-9ac1-074d0135452a">